lib/chef/resource/windows_certificate.rb

```diff
@@ -128,14 +128,14 @@ class WindowsCertificate < Chef::Resource
       end
 
       action :delete, description: "Deletes a certificate." do
-        cert_obj = fetch_cert
+        cert_obj = verify_cert
 
-        if cert_obj
+        if cert_obj == true || cert_obj == "Certificate Has Expired"
```

Comment:
`verify_cert` only returns a `true` or `false`, so this condition can never be met. 

Given that, can we change `cert_obj` name to something like `cert_is_valid`?

Reply:
ooh, good catch. Thanks. Done and done